### PR TITLE
chore(test): migrate prerender-test

### PR DESCRIPTION
**Pull request by christian-bromann**
migrate prerender-test

Some comments on this PR:
- in order to render the pre-rendered content, I inject an iframe into the document
- the iframe source url is the one provided by the application and referenced using Vite's `/@fs/` prefix which allows to access static content
- I am using some Node.js primitives to massage the path to the HTML file - this is obviously not quite supported in WebdriverIO but it injects polyfills for these

### CODE REVIEW

**Review by rwaskiewicz — APPROVED** [review]
One non-blocking ask

File: test/wdio/test-prerender/src/global/util.ts [hunk 1]
```diff
@@ -6,10 +6,10 @@ export function printLifecycle(cmp: string, lifecycle: string) {
   if (Build.isBrowser) {
     const output = document.getElementById(`client-${lifecycle}`);
     elm.textContent = `${cmp} client ${lifecycle}`;
-    output.appendChild(elm);
+    output?.appendChild(elm);
```
[the remark this answers is not in the document]
Comment (christian-bromann) [reply test/wdio/test-prerender/src/global/util.ts]: Not sure why I added it but it doesn't seem to be needed anyway.

**Review by alicewriteswrongs — APPROVED** [review]
overall this looks straightforward! feel free to merge, I had one thought but we could ignore for now

File: test/wdio/tsconfig-prerender.json [hunk 2]
```diff
@@ -30,6 +30,6 @@
     }
   },
   "include": [
-    "src",
+    "test-prerender/src",
```
[the remark this answers is not in the document]
Comment (christian-bromann) [reply test/wdio/tsconfig-prerender.json:33]: good call!

**Pull request merged**